### ISSUE TITLE
.ignoreDependency is not found under onionArchitecture rule

### ISSUE THREAD

**Issue by danilohgds** [TNG/ArchUnit#336]
Hi guys,

I am trying to ignore dependency in onionArchitecture rules, however the .ignoreDependency from examples is not recognized.

I am using latest release: 0.13.1

```java
ArchRule myRule  = onionArchitecture()
				.domainModels("..planning.model..")
				.domainServices("..planning.service..")
				.applicationServices("..configuration..")
				.adapter("rest", "..adapter.api..")
				.adapter("persistence", "..adapter.spi..")

				.ignoreDependency(ApiPort.class, EventPort.class);
```

Example link : https://github.com/TNG/ArchUnit/blob/master/archunit-example/example-junit5/src/test/java/com/tngtech/archunit/exampletest/junit5/OnionArchitectureTest.java

**Comment by hankem** [comment]
You're looking into a part of the `master` branch on https://github.com/TNG/ArchUnit/ from #320 that is indeed not released. 😉

FYI: Examples for the latest release of ArchUnit can be found in https://github.com/TNG/ArchUnit-Examples/ (where [OnionArchitectureTest](https://github.com/TNG/ArchUnit-Examples/blob/master/example-junit5/src/test/java/com/tngtech/archunit/exampletest/junit5/OnionArchitectureTest.java) is still unchanged).

But if you have an urgent use case for `OnionArchitecture.ignoreDependency`, we might be able to convince @codecholeric to release a new version of ArchUnit? 😉

**Comment by danilohgds** [comment]
Well, I am not sure if I would call it urgent 😸 . In my view/understanding of DDD,  the Ports may live in a package of its own(I might be wrong, who knows) , meaning I currently have something like the following:

-domain
-- model
-- ports
-- service

Given that structure, ports end up issuing violations by accessing model, which is why I wanted to give it a try to ignoreDependency, I thought it was part of 0.13.1, thats my bad :)

My issue is simply solved by having ports as an underlying package of the model package, but I wasn't planning on doing that before.

**Comment by ask4gilles** [comment]
Hi @hankem @codecholeric,
Do you have any plans to release this one? Or any advice on a workaround?
Many thanks!

**Comment by hankem** [comment]
The [current version of ArchUnit](https://github.com/TNG/ArchUnit/releases) is `0.14.1` and `OnionArchitecture`'s `ignoreDependency` was released with [ArchUnit 0.14.0](https://github.com/TNG/ArchUnit/releases/tag/v0.14.0). 
I'm sorry; I didn't realize that this issue was still open. Could you try whether upgrading to `0.14.1` works for you (and close the issue if it does)? Thanks!

**Comment by ask4gilles** [comment]
Thanks, works like a charm with `0.14.1`
